src/main/java/io/github/Alathra/AlathranCommands/commands/CommandPlaytime.java

```diff
@@ -0,0 +1,31 @@
+package io.github.Alathra.AlathranCommands.commands;
+
+import com.github.milkdrinkers.colorparser.ColorParser;
+import dev.jorel.commandapi.CommandAPICommand;
+import dev.jorel.commandapi.arguments.PlayerArgument;
+import dev.jorel.commandapi.executors.CommandArguments;
+import io.github.Alathra.AlathranCommands.AlathranCommands;
+import io.github.Alathra.AlathranCommands.utils.MiscCfg;
+import io.github.Alathra.AlathranCommands.utils.PlaytimeChecker;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+
+public class CommandPlaytime {
+
+    public CommandPlaytime(AlathranCommands pl) {
+        new CommandAPICommand("playtime")
+            .withPermission("alathrancommands.playtime")
+            .withOptionalArguments(new PlayerArgument("checkedPlayer"))
```

Comment:
Does `new PlayerArgument` actually allow inputting offline players and does it really cast properly to offline player?

Reply:
This has now been changed to `new OfflinePlayerArgument`.